src/ui/handlers/form-modal-ui-handler.ts

```diff
@@ -58,38 +64,29 @@ export abstract class FormModalUiHandler extends ModalUiHandler {
       this.updateFields(config, hasTitle);
     }
 
-    this.errorMessage = addTextObject(
-      10,
-      (hasTitle ? 31 : 5) + 20 * (config.length - 1) + 16 + this.getButtonTopMargin(),
-      "",
-      TextStyle.TOOLTIP_CONTENT,
-      {
-        fontSize: "42px",
-        wordWrap: { width: 850 },
-      },
-    )
+    const errorMessageY = (hasTitle ? 31 : 5) + 20 * (config.length - 1) + 16 + this.getButtonTopMargin();
+    const errorMessageOptions: Phaser.Types.GameObjects.Text.TextStyle = { fontSize: "42px", wordWrap: { width: 850 } };
+    this.errorMessage = addTextObject(10, errorMessageY, "", TextStyle.TOOLTIP_CONTENT, errorMessageOptions)
       .setColor(getTextColor(TextStyle.SUMMARY_PINK))
       .setShadowColor(getTextColor(TextStyle.SUMMARY_PINK, true))
       .setVisible(false);
     this.modalContainer.add(this.errorMessage);
   }
 
   protected updateFields(fieldsConfig: InputFieldConfig[], hasTitle: boolean) {
-    const inputContainers = (this.inputContainers = new Array(fieldsConfig.length));
-    const inputs = (this.inputs = new Array(fieldsConfig.length));
-    const formLabels = (this.formLabels = new Array(fieldsConfig.length));
+    this.inputContainers = new Array(fieldsConfig.length);
+    this.inputs = new Array(fieldsConfig.length);
+    this.formLabels = new Array(fieldsConfig.length);
     for (const [f, config] of fieldsConfig.entries()) {
+      const labelY = (hasTitle ? 31 : 5) + 20 * f;
       // The Pokédex Scan Window uses width `300` instead of `160` like the other forms
       // Therefore, the label does not need to be shortened
-      const label = addTextObject(
-        10,
-        (hasTitle ? 31 : 5) + 20 * f,
-        config.label.length > 25 && this.getWidth() < 200 ? config.label.slice(0, 20) + "..." : config.label,
-        TextStyle.TOOLTIP_CONTENT,
-      );
+      const labelContent =
+        config.label.length > 25 && this.getWidth() < 200 ? config.label.slice(0, 20) + "..." : config.label;
```

Comment:
```suggestion
      const labelContent = this.getWidth() < 200 ? truncateString(config.label, 25) : config.label;
```
Needs the import manually added